app/src/main/res/xml/pref_notification.xml

```diff
@@ -0,0 +1,19 @@
+<android.support.v7.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <com.kabouzeid.appthemehelper.common.prefs.supportv7.ATEPreferenceCategory android:title="@string/pref_header_notification">
+
+        <com.kabouzeid.appthemehelper.common.prefs.supportv7.ATESwitchPreference
+            android:defaultValue="true"
+            android:key="colored_notification"
+            android:summary="@string/pref_summary_colored_notification"
+            android:title="@string/pref_title_colored_notification" />
```

Comment:
Yeah, I've removed it from the Colors section. I'm not entirely sure which section it should be in though -- I've just moved it to a new "Notification" section for now.